web/templates/article/edit.html.eex

```diff
@@ -10,7 +10,7 @@
 
     <%= link "Back", to: article_path(@conn, :index) %>
     <footer>
-      <p>Have feedback or questions? Contact us</p>
+      <p>Have feedback or questions? <a href="mailto:c​.hanke@de​.edenspiekermann​.com?subject=POSTBot%20Feedback%20or%20Question">Contact us<a/></p>
```

Comment:
@eschaefer either your editor or wherever you pull the address from is inserting some weird chars into the email address
It's showing up in my editor with extra chars.
![screen shot 2016-10-17 at 11 11 54](https://cloud.githubusercontent.com/assets/85859/19431903/91ef2e10-945a-11e6-8a95-432795910f18.png)